workshops/py_Christmas/README.md

```diff
@@ -0,0 +1,380 @@
+
+
+| name | description | author |
+|---|---|---|
+| PyChristmas | Christmas Greetings with turtle graphics | @raz8153 |
+  
+# PyChristmas 
+
+This workshop is meant to help Python newbies try out a fun library called "[turtle](https://docs.python.org/3/library/turtle.html)".
+			
+In this workshop, we make a Christmas greeting using turtle graphics!
+
+[Link to the demo code.](https://repl.it/@raz8153/ThankfulFirsthandScandisk) 
+
+# Getting Started
+
+Let me give a brief idea of what we are going to do. We are going to make a greeting card with a maroon-ish background with a text written on it saying ***"Merry Christmas"***. We are going to add a Christmas tree above the greeting. That is, we need a green tree with a brown stem, a yellow/golden star on top, and tiny red balls on either side of the tree.
+
+![alt completed_output_pic](https://cloud-4ipo9natc.vercel.app/3merry_christmas.png) 
+
+To break it down a bit more, we have made the leaves and stem out of multiple boxes or rectangles, with tiny circles on either end of the rectangle. Finally, a star on top of the center.
+![alt tree_without_color_filled](https://cloud-4ipo9natc.vercel.app/1image2.png)
+
+# Shall we start coding?
+
+Create a new [repl.it](https://repl.it) project and select **Python** as the language from the drop-down menu on the right.
+We are gonna make 2 python files. (Writing on a single page may make it a bit too complicated for a beginner)
```

Comment:
Remove the things in the parentheses—it kinda throws the momentum of the workshop off